### ISSUE TITLE
Standardize chapter-related folder names in */src*

### ISSUE THREAD

**Issue by misicnenad** [NethermindEth/StarknetByExample#211]
### TL;DR
Folder names for chapters should use actual names related to their subject's, like: `getting-started`, `components`, `applications` and `advanced-concepts`.

Folders in question are in */src*: https://github.com/NethermindEth/StarknetByExample/tree/main/src

### Context
Three out of four chapters use `chXX` format for their folder names and then there is one *src/components* folder:
![image](https://github.com/NethermindEth/StarknetByExample/assets/19427053/0740484e-a01b-4820-afc9-1e919c5f101a)

To make matters a bit more confusing, the correct order of the chapters is: `ch00 > components > ch01 > ch02`.

The folder names should be updated in the following way:
- `ch00` -> `getting-started`
- `ch01` -> `applications`
- `ch02` -> `advanced-concepts`

**Comment by raizo07** [comment]
@misicnenad I'll like to work on this

**Comment by misicnenad** [comment]
@raizo07 just create a draft PR and link this issue, I'll review

**Comment by raizo07** [comment]
> @raizo07 just create a draft PR and link this issue, I'll review

Alright, I will.

**Comment by misicnenad** [comment]
Thanks @raizo07, left a comment for you in the PR

**Comment by raizo07** [comment]
> Thanks @raizo07, left a comment for you in the PR

Alright I'll check it out.

**Comment by julio4** [comment]
Completed in #216
